pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -190,7 +195,7 @@ func (r *Reconciler) manageUiComponent(ctx context.Context, th *v1alpha1.TektonH
 
 	th.Status.MarkUiDependenciesInstalled()
 
-	exist, err := checkIfInstallerSetExist(ctx, r.operatorClientSet, version, th, uiInstallerSet)
+	exist, err := r.checkIfInstallerSetExist(ctx, r.operatorClientSet, version, th, uiInstallerSet, ui)
```

Comment:
do we really need both of the last 2 parameters (uiInstallerSet, and ui).
can we pass only uiInstallerSet or ui